### ISSUE TITLE
Live Preview-Only URL

### ISSUE THREAD

**Issue by ghost** [christianalfoni/webpack-bin#36]
I'd love to have a live preview URL in edition to the live edit mode. This would allow you to open the preview on a device. This would pair nicely with some boilerplate for React Material UI. The general idea though is make things friendlier for testing/sharing mobile/touch UIs.

A similar request was also recently made in #33.

**Comment by JaRail** [comment]
Oh, a similar request was also recently made in #33.

**Comment by RamonGebben** [comment]
I think adding the React Material UI should be a different issue.
That issue would not be to include an add button. That way the community and package providers can create the boilerplates.

**Comment by ghost** [comment]
> I think adding the React Material UI should be a different issue.

Absolutely. I was just noting that Material UI could be a nice demo for how a Live Preview on device could be useful. Or perhaps some game engine sample, etc. This issue is only for the main feature, however it's designed/implemented. My example is just to convey some context behind the ask.

**Comment by RamonGebben** [comment]
@christianalfoni I looked over the source which I think takes care of the api/sandbox. Correct me if I'm wrong, it looks like it loads bins on bases of request session. I think the first step would be to change that to accept ID's only. So the iframe would load /api/sandbox/h5Afh2JKg. Then you could just load that in a different tab. 

What do you think?

**Comment by christianalfoni** [comment]
@RamonGebben Thanks for the suggestion and taking time to read the source :) And yeah, you are right, the current bin loaded is attached to the session.

The issue though is that we want the preview window to update automatically when you save your BIN. So we need to add some more logic here. I would suggest to rather have a button that says: "External preview", which fire up a new tab with same url and a "preview" query on it. This would put the BIN in "live mode" and the opened window just opens the same BIN, with the "preview" query on the url. This will make the preview-BIN be shown in preview mode, only displaying the preview window.

Since this will be a new websocket connection it will "join" the live session and get updates as normal.

This way we reuse a lot of logic. Its really just a matter of implementing the preview button and how the BIN should look in preview mode :)

**Comment by Utopiah** [comment]
Yes please make this happen, as mentioned in #33 it would also be useful for webVR testing.

**Comment by christianalfoni** [comment]
I decided to release mobile friendly version first so that you can try it with mobile phone, get "hands on". Please update this issue with thoughts on how "preview" should differ from current mobile version... when it is released, very soon :)
